hack/verify-generated-protobuf.sh

```diff
@@ -19,11 +19,24 @@ set -o nounset
 set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
-source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/build/common.sh"
 
 kube::golang::setup_env
 
-"${KUBE_ROOT}/hack/build-go.sh" cmd/libs/go2idl/go-to-protobuf cmd/libs/go2idl/go-to-protobuf/protoc-gen-gogo
+function prereqs() {
```

Comment:
I vote for removing this function - this is done anyway as part of update-generated-protobuf, so duplicating this logic doesn't make much sense.